components/brave_rewards/resources/tip/components/siteBanner.tsx

```diff
@@ -135,6 +136,11 @@ class Banner extends React.Component<Props, State> {
       logo = `chrome://favicon/size/160@2x/${publisher.logo}`
     }
 
+    let screenName
+    if (this.props.tweetMetaData) {
```

Comment:
could you please move this logic into another function, so that we have like screen name function generator